test/connectors/MessageConnectorSpec.scala

```diff
@@ -131,6 +131,6 @@ object MessageConnectorSpec {
       case _       => matching("""\b[0-9a-f]{8}\b-[0-9a-f]{4}-[0-9a-f]{4}-[0-9a-f]{4}-\b[0-9a-f]{12}\b""")
     }
 
-  private val postUrl                        = "/common-transit-convention-trader-at-destination/message-notification"
+  private val postUrl                        = "/common-transit-convention-trader-at-departure/movements/departures"
```

Comment:
This should probably be "/transits-movements-trader-at-departure-stub/movements/departures"